### PR TITLE
Sphinx indices for ch.bafu.bundesinventare-*

### PR DESCRIPTION
**Pull request by nadine-piveteau**
Tested with `service-sphinxsearch/test/test.py`

Deployment date: 06.12.17

### CODE REVIEW

File: conf/bafu.conf.part [hunk 1]
```diff
@@ -547,18 +547,180 @@ source src_ch_bafu_bundesinventare-auen : def_searchable_features
...
+}
+
+source src_ch_bafu_bundesinventare-auen_anhang2 : def_searchable_features
+{
+    sql_db = bafu_dev
+    sql_query = \
+        SELECT bgdi_id::bigint as id \
+            , name as label \
+            , 'feature' as origin \
+            , objnummer::text||' '||remove_accents(coalesce(name,' ')||' '||coalesce(typ_de,' ')||' '||coalesce(typ_fr,' ')||' '||coalesce(typ_it,' ')) as detail \
```
Comment (loicgasser) [inline conf/bafu.conf.part]: concat_ws is safefer and avoids multiple spaces when an attribute's value is null.
`remove_accent(concat_ws(' ', objnummer::text, name, typ_de, typ_fr, typ_it))`

File: conf/bafu.conf.part [hunk 2]
```diff
@@ -547,18 +547,180 @@ source src_ch_bafu_bundesinventare-auen : def_searchable_features
...
+}
+
+source src_ch_bafu_bundesinventare-amphibien : def_searchable_features
+{
+    sql_db = bafu_dev
+    sql_query = \
+        SELECT bgdi_id::bigint as id \
+            , name as label \
+            , 'feature' as origin \
+            , objnummer::text||' '||remove_accents(coalesce(name,' ')) as detail \
```
Comment (loicgasser) [inline conf/bafu.conf.part]: concat_ws

File: conf/bafu.conf.part [hunk 3]
```diff
@@ -547,18 +547,180 @@ source src_ch_bafu_bundesinventare-auen : def_searchable_features
...
+}
+
+source src_ch_bafu_bundesinventare-amphibien_anhang4 : def_searchable_features
+{
+    sql_db = bafu_dev
+    sql_query = \
+        SELECT bgdi_id::bigint as id \
+            , name as label \
+            , 'feature' as origin \
+            , objnummer::text||' '||remove_accents(coalesce(name,' ')) as detail \
```
Comment (loicgasser) [inline conf/bafu.conf.part]: concat_ws

File: conf/bafu.conf.part [hunk 4]
```diff
@@ -547,18 +547,180 @@ source src_ch_bafu_bundesinventare-auen : def_searchable_features
...
+}
+
+source src_ch_bafu_bundesinventare-amphibien_wanderobjekte : def_searchable_features
+{
+    sql_db = bafu_dev
+    sql_query = \
+        SELECT bgdi_id::bigint as id \
+            , name as label \
+            , 'feature' as origin \
+            , objnummer::text||' '||remove_accents(coalesce(name,' ')) as detail \
```
Comment (loicgasser) [inline conf/bafu.conf.part]: concat_ws

**Comment by nadine-piveteau** [comment]
Thanks for your feedback! It's done and tested :)

**Comment by loicgasser** [comment]
hi but you can't define the same source multiple times, you have to remove the old references first.

**Pull request merged**